demo/html/index.html

```diff
@@ -0,0 +1,67 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <title>Yii Demos</title>
+
+    <!-- Bootstrap core CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+
+    <!-- Favicons -->
+    <link rel="icon" href="/docs/5.0/assets/img/favicons/favicon.ico">
+    <meta name="theme-color" content="#7952b3">
+
+    <style>
+        .bd-placeholder-img {
+            font-size: 1.125rem;
+            text-anchor: middle;
+            -webkit-user-select: none;
+            -moz-user-select: none;
+            user-select: none;
+        }
+
+        @media (min-width: 768px) {
+            .bd-placeholder-img-lg {
+                font-size: 3.5rem;
+            }
+        }
+    </style>
+
+
+    <!-- Custom styles for this template -->
+    <link href="heroes.css" rel="stylesheet">
+</head>
+<body>
+
+<main>
+    <h1 class="visually-hidden">Demo examples</h1>
+
+    <div class="px-4 py-5 my-3 text-center">
+        <img class="d-block mx-auto mb-4" src="https://github.com/yiisoft.png" alt="">
```

Comment:
Use svg format for more quality

Reply:
https://github.com/yiisoft.svg
It doesn't exist

Reply:
https://www.yiiframework.com/image/design/logo/yii3_sign.svg?